### ISSUE TITLE
special attributes in definition lists

### ISSUE THREAD

**Issue by CircleCode** [michelf/php-markdown#108]
I know there is a debate on whether class / ids should be allowed  for each element / block.

Without entering this debate, I'd suggest allowing special attributes for terms in definition lists. this is especially useful with long lists of terms / definitions.

For now, it is required to turn these terms in title to make anchors.

Moreover, visually, these definition lists are quite similar to title, and I'd suggest

```
term {#id, .class}
:   definition
```

what is your feeling about this ? should it be proposed for all implementations in Markdown-Discuss mailing list ?

**Comment by michelf** [comment]
Makes sense as definition terms are often something you want to link to, and this syntax follows the current convention.

Discussing attribute blocks (as I call them in the documentation) on Markdown-Discuss might help popularizing them, or it might not do anything.

**Comment by CircleCode** [comment]
message posted just now to Markdown-discuss… let's see how it will evolve… maybe we will reach a common implementation in major markdown flavors ?
